.travis.yml

```diff
@@ -33,6 +34,17 @@ install:
   - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.0/minikube-linux-amd64
   - sudo chmod +x minikube
   - sudo mv minikube /usr/local/bin/
+  # we need openSeaChest repo to build node-disk-manager
+  - pushd .
+  - cd ..
+  - git clone --recursive https://github.com/openebs/openSeaChest.git 
```

Comment:
it is wise to checkout a specific release tag here, rather than gambling that something upstream doesn't break

Reply:
Your answer doesn't address the problem I raised. Perhaps I wasn't clear. It is wise to explicitly set the tag you want to use when clone git repos and autobuilding.

Reply:
@richardelling If I am not you want build to be done on a specific tag. I am plaing to build seachest library out of Release-18.10 tag. I hope that is inline with your thoughts.